content/blog/kubecon-2023.md

```diff
@@ -0,0 +1,54 @@
+
+### 2. Back to the Future: Next-Generation Cloud Native Security, Andrew Martin & Matt Jarvis
+{{< youtube tPeQQaZcjBs>}}
+
+Last year I quickly became a fan during Andrew Martin's excellent (and slightly chaotic) (talk)[https://youtu.be/1HbwfpE4XKY] at Kubecon Valencia, and ever since then I've made sure to keep an eye on whatever talks are coming from Controlplane.
+
+This year, both he and Matt Jarvis from Snyk gave us a quick history lesson and made some predictions about the near-to-short term future of Cloud Native security, touching on everything from RISC to AI, WASM to Quantum Computing and much, much more.
+
+### 3. Malicious Compliance: Reflections on Trusting Container Scanners, Ian Coldwater, Duffie Cooley, Brad Geesaman, Rory McCune, Datadog
+{{< youtube 9weGi0csBZM>}}
+
+Out of all the Kubecon 2023 talks, this one gave me the most pause. At my current assignment, we're hard at work trying to get our development teams on standardized pipelines with built-in security scanning, making slow but steady progress. 
+
+Seeing how easy it is to fool our current set of container scanners with a couple of trivial additions to Dockerfiles makes me hope that none of those teams come across this talk.
+
+
+There was a bunch of other great talks too, but unfortunately also a couple of talks that really fell flat with me. I'm not writing this to badmouth those people or the quality of the talks, instead I'm just guessing I'm no longer the target audience, as I alluded to earlier.
+
+Kubecon mentioned this year that over half of their audience are first-time attendees. Last year when I was a first-timer, I recall a similar percentage being mentioned. With Kubecon's attendence breaking into the 10.000's, it is one of the largest open source conferences in Europe, so it definitely doing fine, but I'm very much in doubt if I'll be attending KubeCon Paris in 2024.
+
+The keynotes this year were largely uninteresting to me, and some misfortune (broken collarbone the week before) and underestimating (I tried to both participate in the Thursday CTF as well as attend every talk in the morning/afternoon sessions, didn't work out great) made for a generally unremarkable conference experience. It didn't help that most of my (ex)-colleagues attending were local, so nobody felt the need to head out into town and explore Amsterdam, a city most of us are already way too familiar with.
+
+So do I regret my time spent at KubeCon 2023? Definitely not. Will I be considering KubeCon Paris 2024? Probably. Will I be on the lookout for other conferences that align more with my interests? 
+
+Already looking. Suggestions welcome.
```

Comment:
Comment to myself: It doesn't really make sense to ask for suggestions when there's no comment section under the blog...so write a better ending.

Reply:
You can link to your Twitter or something (if you use it)